--- a/src/windows-emulator/syscalls.cpp
+++ b/src/windows-emulator/syscalls.cpp
@@ -310,6 +310,7 @@
                                       uint64_t data, ULONG data_size);
         NTSTATUS handle_NtDeleteValueKey(const syscall_context& c, handle key_handle,
                                          emulator_object<UNICODE_STRING<EmulatorTraits<Emu64>>> value_name);
+        NTSTATUS handle_NtDeleteKey(const syscall_context& c, handle key_handle);
         NTSTATUS handle_NtNotifyChangeKey();
         NTSTATUS handle_NtSetInformationKey(const syscall_context& c, handle key_handle, KEY_SET_INFORMATION_CLASS key_information_class,
                                             uint64_t key_information, ULONG length);
@@ -1458,6 +1459,7 @@
         add_handler(NtCreateKey);
         add_handler(NtSetValueKey);
         add_handler(NtDeleteValueKey);
+        add_handler(NtDeleteKey);
         add_handler(NtNotifyChangeKey);
         add_handler(NtGetCurrentProcessorNumberEx);
         add_handler(NtGetCurrentProcessorNumber);
